lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb

```diff
@@ -135,114 +173,367 @@ def plant_loop_apply_prm_number_of_chillers(plant_loop, sizing_run_dir = nil)
       chiller_compressor_type = 'Centrifugal'
     end
 
-    # Get all existing chillers and pumps
-    chillers = []
-    pumps = []
-    plant_loop.supplyComponents.each do |sc|
-      if sc.to_ChillerElectricEIR.is_initialized
-        chillers << sc.to_ChillerElectricEIR.get
-      elsif sc.to_PumpConstantSpeed.is_initialized
-        pumps << sc.to_PumpConstantSpeed.get
-      elsif sc.to_PumpVariableSpeed.is_initialized
-        pumps << sc.to_PumpVariableSpeed.get
+    if chillers.length > num_chillers
+      OpenStudio.logFree(OpenStudio::Error, 'prm.log', "For #{plant_loop.name}, the existing number of chillers exceeds the recommended amount. We have not accounted for this in the codebase yet.")
+    end
```

Comment:
I'm not sure if understand why this is needed. Could you please clarify?

Reply:
The way I figure, the function does not explicitly prohibit the plant loop that's being passed into the function from having multiple chillers. If the required number of chillers has fewer chillers than what's already in the loop. The code only explicitly adds chillers to the loop-- it does not remove them. The loop below it:

https://github.com/NREL/openstudio-standards/blob/cb46fd1a3bec9c94d637259d2443cc04c44f4884/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb#L186-L224

Note how it only iterates over the number of required chillers, it does not remove any.

Reply:
I see, thanks!